### PR TITLE
Issue #PS-1101 feat: UI to show learners list and related pop ups

### PR DESCRIPTION
**Pull request by suvarnakale**
## Description

Please describe the changes in your pull request in few words here.

### Changes

List the changes done to fix a bug or introducing a new feature.

## How to test

Describe the steps required to test the changes proposed in the pull request.

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)

## Summary by CodeRabbit

- **New Features**
	- Added new translations for various learner and block management actions.
	- Introduced radio button selection in the `ManageCentersModal` for easier user options.
	- Added user management navigation and button in the menu drawer.
	- Implemented `LearnersList` component in the `manageUser` page to handle and display learner data.

- **Enhancements**
	- Improved `ConfirmationModal` with optional `handleAction` to better control modal actions.
	- Updated center assignment logic in `LearnersList` for more precise management.
	- Enhanced UI to reflect new learner and block management features.

### CODE REVIEW

File: src/components/ConfirmationModal.tsx [hunk 1]
```diff
@@ -88,8 +88,12 @@ const ConfirmationModal: React.FC<ConfirmationModalProps> = ({
             variant="contained"
             color="primary"
             onClick={() => {
-              handleAction();
-              handleCloseModel();
+              if (handleAction !== undefined) {
+                handleAction();
+                handleCloseModel();
```
Comment (itsvick) [inline src/components/ConfirmationModal.tsx:93]: @suvarnakale rename it to `handleCloseModal`

File: src/components/ManageCentersModal.tsx [hunk 2]
```diff
@@ -152,12 +166,22 @@ const ManageCentersModal: React.FC<ManageUsersModalProps> = ({
...
-                        <Checkbox
-                          sx={{ pb: '20px' }}
-                          className="checkBox_svg"
-                          checked={checkedCenters.includes(name)}
-                          onChange={() => handleToggle(name)}
-                        />
+                        {isForLearner ? (
+                          <Radio
+                            sx={{ pb: '20px' }}
+                            className="radio_svg"
```
Comment (itsvick) [inline src/components/ManageCentersModal.tsx:172]: @suvarnakale please rename the class. It should not be a snake case

File: src/pages/manageUser.tsx [hunk 3]
```diff
@@ -611,6 +664,87 @@ const manageUsers = () => {
...
+                    </Select>
+                  </FormControl>
+                </Box>
+              </Grid>
+            </Grid>
+            {learnerData?.map((data: any) => (
+              <LearnersList
+                key={data.userId}
+                learnerName={data.name}
+                isDropout={data.memberStatus === 'dropout'}
```
Comment (itsvick) [inline src/pages/manageUser.tsx:724]: @suvarnakale rushikesh has added new constants for dropout in the constant.ts file please use same here

File: src/components/ManageCentersModal.tsx (hunk 2 above)
Comment (suvarnakale) [reply src/components/ManageCentersModal.tsx:172]: sure.

File: src/pages/manageUser.tsx (hunk 3 above)
Comment (suvarnakale) [reply src/pages/manageUser.tsx:724]: sure

**Pull request merged**